### PR TITLE
13-pass-conf

### PR DESCRIPTION
**Pull request by cbmckni**

### CODE REVIEW

File: controller/Dockerfile [hunk 1]
```diff
@@ -42,6 +42,8 @@ RUN rm -f /packages/slurm-*.rpm /packages/openmpi-*.rpm \
   && rm -rf /var/cache/yum \
   && rm -f /Lmod-7.7.tar.bz2
 
-COPY docker-entrypoint.sh /docker-entrypoint.sh
+#Passing conf file
+COPY docker-entrypoint.sh slur*.conf ./
```
[the remark this answers is not in the document]
Comment (cbmckni) [reply controller/Dockerfile]: Good idea. Done.

**Comment by cbmckni** [comment]
Ready to merge?

**Comment by cbmckni** [comment]
> Would suggest using a subdir of an exiting mount. 
Do you mean something along the lines of:
```
volumes:
      - home:/home
      - secret:/.secret
      - ./slurm-conf:/home/.config
```
Since the source for the "home" volume is ```/var/lib/docker/volumes/slurmindocker_home/_data```, I think having a mount in ```slurm-in-docker/``` would be a good idea, and ```slurm-in-docker/slurm-conf``` seems like an intuitive location to place a slurm.conf file.

**Comment by mjstealey** [comment]
replaced by #19 - closing

**Pull request closed without merging**